components/tabs/android/src/main/java/mozilla/appservices/tabs/DeviceType.kt

```diff
@@ -3,9 +3,5 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 @file:Suppress("InvalidPackageDeclaration")
+// XXX - do we need this?
```

Comment:
No, this file is unnecessary now. It was created to prevent this renaming from being a breaking change in android so without they breaking change this file has no purpose. 🎉